### ISSUE TITLE
After upgrade, osxphotos export --directory template produces error when "-" or other characters are included

### ISSUE THREAD

**Issue by danielsbrewer** [RhetTbull/osxphotos#634]
**Describe the bug**
Today I upgraded from  0.44.13 to 0.45.11. 
Now if I use "--directory '{folder_album( - )}'" an error occurs "Error: Invalid template '{folder_album( - )}': SyntaxError: None:1:16: error: Expected ')' at position (1, 16) => 'er_album( *- )}'."
It worked fine in the previous version.
It works fine if I use "--directory '{folder_album( )}'" (one space).
The same error occurs if I use "--directory '{folder_album( aaa )}'".
The same error occurs if I use "--directory '{folder_album(  )}'" (two spaces).
The same error occurs if I use "--directory "{folder_album(' - ')}"" (single quote withing the brackets).

**To Reproduce**
1. What' the full command line you used with osxphotos?
`osxphotos export --folder "2017" --touch-file --skip-original-if-edited --exiftool  --exiftool-merge-keywords  --exiftool-merge-persons --person-keyword --album-keyword --directory "{folder_album( - )}" osxphotos_Export/`
2. What was the error output?
```
Exporting 2937 photos to /Users/nsu13cbu/Desktop/osxphotos_Export...
  [------------------------------------]  0/2937
Usage: osxphotos export [OPTIONS] [PHOTOS_LIBRARY]... DEST
Try 'osxphotos export -h' for help.

Error: Invalid template '{folder_album( - )}': SyntaxError: None:1:16: error: Expected ')' at position (1, 16) => 'er_album( *- )}'.
```

**Expected behavior**
Production of directories with "folder name - album name"

**Desktop (please complete the following information):**
 - OS: macOS 11.6.2
 - osxphotos version: 0.45.11

**Comment by RhetTbull** [comment]
I am not able to reproduce this:

```
[I] ➜ osxphotos --version
osxphotos, version 0.44.13
[I] ➜ osxphotos export ~/Desktop/export  --directory '{folder_album( - )}'
Using last opened Photos library: /Users/rhet/Pictures/Test-10.15.7.photoslibrary
Exporting 28 photos to /Users/rhet/Desktop/export...
  [------------------------------------]    0%
Usage: osxphotos export [OPTIONS] [PHOTOS_LIBRARY]... DEST
Try 'osxphotos export -h' for help.

Error: Invalid template '{folder_album( - )}': SyntaxError: None:1:16: error: Expected ')' at position (1, 16) => 'er_album( *- )}'.
```

It's reasonable to join folders/albums with more than one character so I'll see if the template engine can accommodate this.

**Comment by RhetTbull** [comment]
Here's the actual parser grammar for the path separator (the feature in question) and it allows only 0 or 1 character so it was a bug if this was every working for you.

```
PathSep:
    (
    "("
    (value=/[^\(\)\{\}]{0,1}/)?
    ")"
    )?
;
```

**Comment by danielsbrewer** [comment]
Just had a thought that I had upgraded my OS inbetween so that might have something to do with it as well. Not a deal breaker as I can always rename later.

**Comment by RhetTbull** [comment]
I've got a fix in work!

**Comment by RhetTbull** [comment]
This should be fixed in [v0.45.12](https://github.com/RhetTbull/osxphotos/releases/tag/v0.45.12).  It wasn't a bug but was acting as designed so I'm surprised it ever worked for you--I can see no way that should have worked. But, the path_sep field (the part in between the `(` `)`) can now support multiple characters.

**Comment by danielsbrewer** [comment]
I've just tried it and that it working great. Thanks for fixing. I have no clue how it was doing it before, sorry, but it was some how. Thanks again for sorting out.
